packages/protocol/contracts/governance/Proposals.sol

```diff
@@ -224,33 +224,6 @@ library Proposals {
     return FixidityLib.newFixedFraction(yesVotes, yesVotes.add(noVotes));
   }
 
-  /**
-   * @notice Returns the stage of a dequeued proposal.
-   * @param proposal The proposal struct.
-   * @param stageDurations The durations of the dequeued proposal stages.
-   * @return The stage of the dequeued proposal.
-   * @dev Must be called on a dequeued proposal.
-   */
-  function getDequeuedStage(Proposal storage proposal, StageDurations storage stageDurations)
```

Comment:
Removing this would be considered a breaking change, contrackit will have to be updated. Any reason you couldn't write the new logic here?

Reply:
yeah you're right, it's internal now I'm seeing